src/granite/query/builder.cr

```diff
@@ -88,6 +93,12 @@ class Granite::Query::Builder(Model)
     self
   end
 
+  def and(stmt : String, value : Granite::Columns::Type)
+    @where_fields << {join: :and, stmt: stmt, value: value}
+
+    self
+  end
+
   def or(**matches)
```

Comment:
Should probably also support `OR` clauses.

Reply:
Ah, right. Done.

Reply:
Getting some specs on it as well would be :100:.

Reply:
I added a spec for `#where`, but I didn't see any existing specs for `#and` and `#or`, so wasn't sure what was appropriate. I can try throwing some up though.